barretenberg/cpp/src/barretenberg/dsl/acir_format/multi_scalar_mul.cpp

```diff
@@ -34,7 +34,7 @@ template <typename Builder> void create_multi_scalar_mul_constraint(Builder& bui
     }
 
     // Call batch_mul to multiply the points and scalars and sum the results
-    auto output_point = cycle_group_ct::batch_mul(scalars, points);
+    auto output_point = cycle_group_ct::batch_mul(points, scalars);
```

Comment:
the signature of batch_mul in cycle_group was different from native and biggroup (scalars and points order was switched) so I modified it to be consistent.